pkg/operator/ceph/object/health.go

```diff
@@ -152,17 +146,9 @@ func (c *bucketChecker) checkObjectStoreHealth() error {
 	s3SecretKey = *user.SecretKey
 
 	if c.objectStoreSpec.IsTLSEnabled() {
-		if c.objectStoreSpec.Gateway.SSLCertificateRef != "" {
-			tlsSecretCert, err = c.context.Clientset.CoreV1().Secrets(c.namespacedName.Namespace).Get(ctx, c.objectStoreSpec.Gateway.SSLCertificateRef, metav1.GetOptions{})
-			if err != nil {
-				return errors.Wrapf(err, "failed to get %q secret containing TLS certificate", c.objectStoreSpec.Gateway.SSLCertificateRef)
-			}
-			tlsCert = tlsSecretCert.Data[certKeyName]
-		} else if c.objectStoreSpec.GetServiceServingCert() != "" {
-			tlsCert, err = ioutil.ReadFile(ServiceServingCertCAFile)
-			if err != nil {
-				return errors.Wrapf(err, "failed to fetch TLS certificate from %q", ServiceServingCertCAFile)
-			}
+		tlsCert, err = GetTLSCACert(c.objContext, c.objectStoreSpec)
+		if err != nil {
+			return errors.Wrap(err, "failed to fetch cacert for TLS connection")
```

Comment:
2 things:
1. CA cert is two words
2. Is there more info you can add to this error to identify it more clearly? E.g., the object store name, user name, or things like that?

```suggestion
		return errors.Wrap(err, "failed to fetch CA cert for TLS connection")
```